src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -172,34 +168,4 @@ private static DetailNode parseFile(File file) throws IOException {
             System.getProperty("file.encoding", "UTF-8"));
         return parseJavadocAsDetailNode(text.getFullText().toString());
     }
-
-    /**
-     * Creates DetailAST block comment to pass it to the Javadoc parser.
-     * @param content comment content.
-     * @return DetailAST block comment
-     */
-    private static DetailAST createFakeBlockComment(String content) {
```

Comment:
This change results in shippable failure for profile `-Ppitest-checkstyle-tree-walker,no-validations`. This apparently happens because of decreased line count in `DetailNodeTreeStringPrinter` class. As can be seen the surviving mutation is same between [master pit-test reports](https://master-pitreports.netlify.com/) and [patch pit test reports](https://3311-pitreports.netlify.com/) and all the new mutations are being killed